perception/radar_fusion_to_detected_object/src/radar_fusion_to_detected_object.cpp

```diff
@@ -121,6 +124,18 @@ RadarFusionToDetectedObject::Output RadarFusionToDetectedObject::update(
   return output;
 }
 
+// Judge whether twist covariance is available.
+bool RadarFusionToDetectedObject::hasTwistCovariance(
+  const TwistWithCovariance & twist_with_covariance)
+{
+  auto covariance = twist_with_covariance.covariance;
+  if (covariance.at(0) == 0.0 && covariance.at(7) == 0.0 && covariance.at(14) == 0.0) {
```

Comment:
[imo] If you can avoid using magic number here, you should use enum like [this](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/multi_object_tracker/include/multi_object_tracker/utils/utils.hpp#L36-L73)

Reply:
fixed at [229d15a](https://github.com/autowarefoundation/autoware.universe/pull/1692/commits/229d15a462321b2b3bd88b018c741ce4801b1d8a)